apps/console/src/extensions/components/logs/api/logs.ts

```diff
@@ -0,0 +1,47 @@
+/**
```

Comment:
rename the file to `useLogs .ts`

Reply:
Fixed with Fixed with f159ffb40208f9d92ad1a9fb1965e1609baf6862